stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/options/AttachmentGalleryOptionsViewStyle.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.gallery.options
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import androidx.annotation.ColorInt
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.TransformStyle
+import io.getstream.chat.android.ui.common.extensions.internal.getColorCompat
+import io.getstream.chat.android.ui.common.extensions.internal.getDrawableCompat
+import io.getstream.chat.android.ui.common.extensions.internal.use
+import io.getstream.chat.android.ui.gallery.options.internal.AttachmentGalleryOptionsView
+
+/**
+ * Style for [AttachmentGalleryOptionsView].
+ *
+ * @param replyOptionEnabled If the "reply" option is present in the list.
+ * @param replyOptionDrawable  The icon to the "reply" option.
+ * @param showInChatOptionEnabled If the "show in chat" option present in the list.
+ * @param showInChatOptionDrawable The icon for the "show in chat" option.
+ * @param saveImageOptionEnabled If the "save image" option is present in the list.
+ * @param saveImageOptionDrawable The icon for the "save image" option.
+ * @param deleteOptionEnabled If the "delete" option is present in the list.
+ * @param deleteOptionDrawable The icon for the "delete" option.
+ * @param deleteOptionTextColor The text color of the "delete" option.
+ */
+public data class AttachmentGalleryOptionsViewStyle(
+    val replyOptionEnabled: Boolean,
+    val replyOptionDrawable: Drawable,
+    val showInChatOptionEnabled: Boolean,
+    val showInChatOptionDrawable: Drawable,
+    val saveImageOptionEnabled: Boolean,
+    val saveImageOptionDrawable: Drawable,
+    val deleteOptionEnabled: Boolean,
+    val deleteOptionDrawable: Drawable,
+    @ColorInt val deleteOptionTextColor: Int,
```

Comment:
You could add the background colour and option TextStyle as well if we are exposing customisation options.